src/Homework 3/Program.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using ToDoApp;
+
+namespace ConsoleApp1
+{
+    class ToDoApp
+    {
+        static void Main(string[] args)
+        {
+            var organizer = new Organizer();
+            bool stop = false;
+            string action;
+            do
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Press \n1 - to add a task, \n2 - to look throw my tasks, \n3 - to update a task, \n4 - to cancel a task, \n0 - to exit \n");
+                Console.ResetColor();
+                action = Console.ReadLine();
+                Console.WriteLine();
+                switch (action)
+                {
+                    case "1":
+                        AddTaskToOrg(organizer);
+                        break;
+                    case "2":
+                        ToLookThrow(organizer);
+                        break;
+                    case "3":
+                        UpdateTask(organizer);
+                        break;
+                    case "4":
+                        {
+                            Console.WriteLine("Введите ID номер задачи:");
+                            int id = Convert.ToInt32(Console.ReadLine());
+                            organizer.CancelTask(id);
+
+                        }
+                        break;
+                    case "0":
+                        stop = true;
+                        break;
+                    default:
+                        Console.WriteLine("Enter correct number:");
+                        break;
+                }
+
+
+            } while (!stop);
+        }
+
+        private static void AddTaskToOrg(Organizer organizer)
+        {
+            Console.WriteLine("Введите описание задачи");
+            var description = Console.ReadLine();
+            Console.WriteLine("Введите дату начала задачи");
+
+            DateTime start;
+            do
+            {
+                try
+                {
+                    start = Convert.ToDateTime(Console.ReadLine());
+                }
+                catch
+                {
+                    Console.WriteLine("Введите дату в формате ДД/ММ/ГГГГ");
+                    continue;
+                }
+
+                break;
+            } while (true);
+
+            Console.WriteLine("Введите дату окончания задачи");
+            DateTime end;
+            do
+            {
+                try
+                {
+                    end = Convert.ToDateTime(Console.ReadLine());
+                }
+                catch
+                {
+                    Console.WriteLine("Введите дату в формате ДД/ММ/ГГГГ");
+                    continue;
+                }
+
+                break;
+            } while (true);
+
+            organizer.AddTask(start, end, description);
+        }
+        public static void ToLookThrow(Organizer organizer)
```

Comment:
For now, normally, naming is not a big issue, but here it is completely impossible to know what method intended to do )

Reply:
ToLookThrow - to overview
I agree - It is very difficult to get what i meant for those who might read the code)